catalyst/dl/callbacks/checkpoint.py

```diff
@@ -128,15 +128,20 @@ def load_checkpoint(*, filename, state: RunnerState):
             raise Exception(f"No checkpoint found at {filename}")
 
     def get_metric(self, last_valid_metrics) -> Dict:
-        checkpoints = [
+        top_best_checkpoints = [
             (Path(filepath).stem, valid_metric)
             for (filepath, _, valid_metric) in self.top_best_metrics
         ]
-        best_valid_metrics = checkpoints[0][1]
+        all_checkpoint_metrics = [
+            (f"chekpoint_{order_index}", valid_metric)
```

Comment:
maybe, it's better to name `epoch_{i}` metric?

Reply:
Yes, it is better one.